demo/rag/README.md

```diff
@@ -0,0 +1,133 @@
+| EMBEDDING_FUNCTION_TYPE            | Embedding function type                             | `default` (options: `api`, `local`, `default`)   | Required                               |
+| OPENAI_API_KEY                     | LLM API Key (supports OpenAI, Qwen, etc.)           | Must be set                                       | Required (for generating answers)      |
+| OPENAI_BASE_URL                    | LLM API base URL                                     | https://dashscope.aliyuncs.com/compatible-mode/v1 | Optional                               |
+| OPENAI_MODEL_NAME                  | Language model name                                  | qwen-plus                                        | Optional                               |
+| EMBEDDING_API_KEY                  | Embedding API Key                                    | -                                                 | Required when `EMBEDDING_FUNCTION_TYPE=api` |
+| EMBEDDING_BASE_URL                 | Embedding API base URL                               | https://dashscope.aliyuncs.com/compatible-mode/v1 | Optional when `EMBEDDING_FUNCTION_TYPE=api` |
+| EMBEDDING_MODEL_NAME               | Embedding model name                                | text-embedding-v4                                | Required when `EMBEDDING_FUNCTION_TYPE=api` |
+| SENTENCE_TRANSFORMERS_MODEL_NAME   | Local model name                                     | all-mpnet-base-v2                               | Optional when `EMBEDDING_FUNCTION_TYPE=local` |
+| SENTENCE_TRANSFORMERS_DEVICE       | Device to run on                                     | cpu                                              | Optional when `EMBEDDING_FUNCTION_TYPE=local` |
+| SEEKDB_DIR                         | seekdb database directory                           | ./data/seekdb_rag                                | Optional                               |
+| SEEKDB_NAME                        | Database name                                        | test                                             | Optional                               |
+| COLLECTION_NAME                    | Collection name                                     | embeddings                                       | Optional                               |
+
+> **Tip:** 
+> - If using `default` type, only configure `EMBEDDING_FUNCTION_TYPE=default` and LLM-related settings
+> - If using `api` type, additional Embedding API variables need to be configured
+> - If using `local` type, install the `sentence-transformers` library and optionally configure the model name
+
+### 2. Prepare Data
+
+We use pyseekdb's SDK documentation as an example. You can also use your own Markdown documents.
+
+**Import Data:**
+
+Run the data import script:
+
+```bash
+uv run python seekdb_insert.py ../../README.md
```

Comment:
seekdb_insert.py support add one directory, add one command to add one directory